### PR TITLE
Component Design: Search Panel

### PR DESCRIPTION
**Pull request by nirav-improwised**
Created search panel, added background image to it.

### CODE REVIEW

File: pages/index.vue [hunk 1]
```diff
@@ -1,4 +1,13 @@
 <template>
     <div>
+        <SearchPanel />
     </div>
 </template>
+
+<style>
+main{ 
+    background: linear-gradient(rgba(25, 25, 25, 0.7),
+                    rgba(25, 25, 25, 0.7)), url("../assets/background.jpg")no-repeat center center/cover;
```
Comment (monikagorkhiya) [inline pages/index.vue]: don't use style in vue page make it a different file.You can't give style main tag,you can create class and in that add style

Comment (nirav-improwised) [reply pages/index.vue]: Corrected..

**Review by monikagorkhiya — APPROVED** [review]

**Review by monikagorkhiya — COMMENTED** [review]
Take care of the responsive layout. you defined CSS which is already provided by bootstrap in built class 
https://getbootstrap.com/docs/5.0/utilities/sizing/

E.g. for width we can use bootstrap built-in because it occurs responsive issues and then we need to do number of lines code

**Comment by nirav-improwised** [comment]
Done. Corrected

**Review by monikagorkhiya — APPROVED** [review]

**Pull request merged**